presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/MetastoreUtil.java

```diff
@@ -441,7 +441,7 @@ else if (type instanceof TinyintType
         for (HiveColumnHandle partitionKey : partitionKeys) {
             String name = partitionKey.getName();
             Domain domain = effectivePredicate.getDomains().get().get(partitionKey);
-            if (domain != null && domain.isNullableSingleValue()) {
+            if (domain != null) {
```

Comment:
What's the issue? Add some details to the commit and a test.

Reply:
I'll update the description and add a test for this method. There's no good way to test this end to end, as it's basically a side effect (the metastore returns the result regardless as the filter is just a hint).